src/developer_config.yaml

```diff
@@ -11,6 +12,7 @@ runlog_writer:
 paths:
   logs_foldername: "testing_pydoop"
   snapshot_path: "/ons/rdbe_dev/snapshot-202012-002-fba5c4ba-fb8c-4a62-87bb-66c725eea5fd.json"
+  masterlist_path: "data/external/ONSPD_NOV_2022_UK.csv"
```

Comment:
Added path to masterlist since it was missing

Reply:
Great